CONTRIBUTING.md

```diff
@@ -398,7 +398,7 @@ in a DC/OS Service by looking at
 
 ### Writing Integration Tests
 
-Writing unit testing can be harder than unit tests, we recommend following best 
```

Comment:
"Writing docs without drinking coffee is harder that writing docs without drinking coffee"